common/src/main/java/tachyon/Version.java

```diff
@@ -25,7 +25,7 @@
 
   static {
     TachyonConf tachyonConf = new TachyonConf();
-    VERSION = tachyonConf.get(Constants.TACHYON_VERSION, "UNDEFINED");
+    VERSION = tachyonConf.get(Constants.TACHYON_VERSION);
```

Comment:
For consistency, shall we always have precondition checking of key's existence ?